EventFilter/L1TRawToDigi/plugins/implementations_stage2/EtSumUnpacker.cc

```diff
@@ -91,6 +91,15 @@ namespace stage2 {
 
        res_->push_back(bx,ht);
 
+
+       //HI-SUM
+
+       l1t::EtSum towCount = l1t::EtSum();
```

Comment:
Yes.  All the previous data do not have kTowerCount types in EtSum.  The unpacker will then assign "0" Pt to TowerCount sums in those cases.   Same as before, when we added new EtSum types.